actions/templates/post/index.js

```diff
@@ -98,7 +104,7 @@ async function main(params) {
       };
     }
 
-    const template = await addTemplate(templateName, githubRepoUrl, params.ACCESS_TOKEN_GITHUB, params.TEMPLATE_REGISTRY_ORG, params.TEMPLATE_REGISTRY_REPOSITORY, `Add "${templateName}" via API`);
+    const template = await addTemplate(dbParams, templateName, githubRepoUrl);
     const issueNumber = await createReviewIssue(templateName, githubRepoUrl, params.ACCESS_TOKEN_GITHUB, params.TEMPLATE_REGISTRY_ORG, params.TEMPLATE_REGISTRY_REPOSITORY);
```

Comment:
Let's comment out creating the review issue and adding the `review` field below for now until we start supporting App Builder templates